src/site/content/en/blog/native-file-system/index.md

```diff
@@ -246,27 +312,32 @@ butDir.addEventListener('click', async (e) => {
 });
 ```
 
-## What's currently supported? {: #whats-supported }
-
-We're still working on some of the implementation for the
-Native File System API, and not everything in the [spec][spec]
-(or [explainer][explainer]) has been completed.
-
-As of Chrome 78, the following functionality is not available, or
-doesn't match the spec:
-
-* Handles are not serializable, meaning they cannot be passed via
-  `postMessage()`, or stored in IndexedDB.
-* Non-atomic writes (i.e. calls to `FileSystemFileHandle.createWriter()`
-  with `inPlace: true`).
-* Writing to a file using a [`WritableStream`][writablestream].
-* The [`FileSystemDirectoryHandle.resolve()`][fs-dir-handle] method.
+## What's new/changed? {: #whats-new }
+
+Chrome 82
+
+* Support for [writable streams](#save-to-disk) was added, and the previous
+  method for writing to disk (`FileSystemWriter`) was removed.
+* File handles can now be serialized and stored in IndexedDB, or sent via
+  `postMessage` to other windows or workers within the same origin. Note that
+  permissions are not retained between browser sessions. For example, when a
+  browser tab is re-opened, and a file handle is obtained from IndexedDB, the
+  user will need to grant permission to read/write to the file again.
+* Added support for [`isSameEntry()`][spec-issameentry], which returns `true`
```

Comment:
A best practice is for bulleted list to follow the same grammatical structure.

Reply:
Suggestions on how to fix this one?

Reply:
I'm not sure why it's not showing against this page, but I've done a PR: https://github.com/GoogleChrome/web.dev/pull/2569/files

Reply:
AWESOME - that makes more sense now, got it. I fixed all the but permissions one.